documentation/API_documentation/home-devices-qod-API-Readiness-Checklist.md

```diff
@@ -0,0 +1,27 @@
+# API Readiness Checklist
+
+Checklist for home-devices-qod 0.4.0-rc.1 in r1.1
+
+| Nr | API release assets  | alpha | release-candidate |  initial<br>public | stable<br> public | Status | Comments |
+|----|----------------------------------------------|:-----:|:-----------------:|:-------:|:------:|:----:|----|
+|  1 | API definition                               |   M   |         M         |    M    |    M   |   Y  | [home_devices_qod.yaml](../../code/API_definitions/home_devices_qod.yaml) |
+|  2 | Design guidelines from Commonalities applied |   O   |         M         |    M    |    M   |   Y  |      |
+|  3 | Guidelines from ICM applied                  |   O   |         M         |    M    |    M   |   Y  |      |
+|  4 | API versioning convention applied            |   M   |         M         |    M    |    M   |   Y  |      |
+|  5 | API documentation                            |   M   |         M         |    M    |    M   |   Y  | Embedded in [API definition](../../code/API_definitions/home_devices_qod.yaml) |
+|  6 | User stories                                 |   O   |         O         |    O    |    M   |   Y  | [home_devices_qod_user_stories.md](./home_devices_qod_user_stories.md) |
+|  7 | Basic API test cases & documentation         |   O   |         M         |    M    |    M   |   Y  | [home-devices-qod.feature](../../code/Test_definitions/home-devices-qod.feature) |
+|  8 | Enhanced API test cases & documentation      |   O   |         O         |    O    |    M   |   Y  | [home-devices-qod.feature](../../code/Test_definitions/home-devices-qod.feature) |
+|  9 | Test result statement                        |   O   |         O         |    O    |    M   |   N  |      |
+| 10 | API release numbering convention applied     |   M   |         M         |    M    |    M   |   Y  |      |
+| 11 | Change log updated                           |   M   |         M         |    M    |    M   |   Y  | [CHANGELOG.md](../../CHANGELOG.md) |
```

Comment:
It's completely correct, but relativ to the root of the repository would work as well :-)

```suggestion
| 11 | Change log updated                           |   M   |         M         |    M    |    M   |   Y  | [CHANGELOG.md](/CHANGELOG.md) |
```